react/components/AutocompleteInput/SearchInput/index.tsx

```diff
@@ -74,7 +81,7 @@ const SearchInput: React.FC<PropTypes.InferProps<typeof propTypes> &
 
   const buttonClasses = classNames(
     activeClass,
-    'bg-base br2 br--right h-regular w3 bw1 ba pa0 bl-0',
+    `bg-base br2 br--right w3 bw1 ba pa0 bl-0' h-${size}`,
```

Comment:
You need to check if the size is valid to prevent a broken style, if it's not one of `small`, `regular` or `large` change it to `regular`

Reply:
Just `size='regular'` as default prop is enough, IMO. :)

Reply:
But what if the component receives a size that doesn't exist? Example `size="aa"`

Reply:
when we a have a non-existent size passed to the prop, we get this behaviour: 
![image](https://user-images.githubusercontent.com/29438497/85164200-658fed80-b23a-11ea-9710-771589e6bbaa.png)

the bar gets smaller that the small sized bar, and we also get a warn about the invalid prop:

![image](https://user-images.githubusercontent.com/29438497/85164176-5a3cc200-b23a-11ea-9f03-23a857843dcc.png)

From the development perspective, isn't better for the developer to see that something might be wrong on the console instead of having that wrong value being replaced by the `regular` size? I'm thinking of something like, If he/she passed a typo, for example, we would replace it for regular and it would be warned anywhere as we're fixing it behind the scenes.

@thayannevls @matheusps

Reply:
I agree with Eden

Reply:
I agree that must have the warning in the console, but not to let the component have a random size like it is now :thinking: other components in styleguide when receiving an invalid size, it forces to go to the `regular` and throws a warning in the console.

Reply:
I agree @edenalsant

Reply:
My only concern is that the component is getting smaller than the `small` size when something is wrong

Reply:
but wouldn't it be a indication that something is wrong?

Reply:
I don't think that getting the component to have a random size is a good indication when something is wrong, you have the error in the console for this. You can check what other components with sizes in styleguide to compare

Reply:
done ✨  @thayannevls

Reply:
@edenalsant. I've discussed with @thayannevls thought a little bit more about this, and she's right. The ideal way to handle this is not just to fill the `height-x` className, but a function that does not allow you to have `height-undefined`. But, you're maybe wondering "it is never undefined" 🤔. In the way you're doing right now, kinda is! The `height-wrongWord` is not defined by the design language, so you render an input with a wrong look (I didn't notice that the inputs had different heights on your print, sorry about that).

Reply:
awesome! thank you, guys! you rock! ✨